### PR TITLE
Added terms and conditions

### PR DESCRIPTION
**Pull request by amir-qayyum-khan**
#### What are the relevant tickets?
fixes https://github.com/mitodl/bootcamp-ecommerce/issues/127

#### What's this PR do?
Add terms and condition page in bootcamp

#### How should this be manually tested?
go to payment page and click on `MIT Bootcamps Terms and Conditions`

@pdpinch 

#### Screenshots (if appropriate)
screen shot 2017-10-09 at 5 51 10 pm

### CODE REVIEW

**Comment by amir-qayyum-khan** [comment]
@noisecapella i am getting this on osx any idea?
```js
This usually happens because your environment has changed since running `npm install`.
watch_1   | Run `npm rebuild node-sass` to build the binding for your current environment.
watch_1   |     at module.exports (/src/node_modules/node-sass/lib/binding.js:15:13)
watch_1   |     at Object.<anonymous> (/src/node_modules/node-sass/lib/index.js:14:35)
watch_1   |     at Module._compile (module.js:541:32)
watch_1   |     at Object.Module._extensions..js (module.js:550:10)
watch_1   |     at Module.load (module.js:458:32)
watch_1   |     at tryModuleLoad (module.js:417:12)
watch_1   |     at Function.Module._load (module.js:409:3)
watch_1   |     at Module.require (module.js:468:17)
watch_1   |     at require (internal/module.js:20:19)
watch_1   |     at Object.<anonymous> (/src/node_modules/sass-loader/index.js:4:12)
watch_1   |  @ ./static/scss/layout.scss 4:14-162 13:2-17:4 14:20-168
watch_1   |  @ ./static/js/entry/style.js
watch_1   |  @ multi ./static/js/entry/style webpack-hot-middleware/client?path=http://0.0.0.0:8098/__webpack_hmr&timeout=20000&reload=true
```

**Comment by noisecapella** [comment]
The color of the link doesn't match the mockup:
screenshot from 2017-10-12 17-57-26

**Comment by noisecapella** [comment]
@amir-qayyum-khan Yarn needs to be updated. Can you update the yarn version in `package.json` to `1.2.1`, then rebuild your docker container and update your local yarn instance via `./scripts/install_yarn.sh`

File: static/scss/_shared.scss [hunk 1]
```diff
@@ -43,4 +43,8 @@ h3.section-header {
   li ul {
     margin-top: $text-page-margin;
   }
+
+  ol-sub-lists {
```
Comment (noisecapella) [inline static/scss/_shared.scss]: This needs a `.` to apply to classes

File: static/scss/payment.scss [hunk 2]
```diff
@@ -19,6 +19,26 @@
     padding-top: 15px;
   }
 
+  .tac {
+    color: rgba(255, 255, 255, .7);
```
Comment (noisecapella) [inline static/scss/payment.scss]: Can you this color into a variable and use it elsewhere within this file?

Comment (amir-qayyum-khan) [reply static/scss/payment.scss]: Done @noisecapella, found existing variable

File: bootcamp/templates/bootcamp/tac.html [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+  month before the start of the program. In the event that there is a difference between the payment due dates
+  specified above and the admissions letter, we will follow the payment due date as indicated on the admissions letter.
+  </p>
+ <strong>Cancellation and Deferral Policies</strong>
+ <p>
+     All registrations made under this agreement will be subject to MIT Bootcamp’s standard cancellation and deferral policies, including:
+ </p>
+ <ol>
+    <li>
+      All cancellations must be made in writing to bootcamp@mit.edu. Cancellation fees are as follows:
```
Comment (noisecapella) [inline bootcamp/templates/bootcamp/tac.html]: Can you make `bootcamp@mit.edu` a mailto link?

File: bootcamp/templates/bootcamp/tac.html [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+      <ol type="a" class="ol-sub-lists">
+          <li>56 days or more prior to the program start date: $250 administrative fee</li>
+          <li>55-22 days prior to the program start date: 50% cancellation fee</li>
+          <li>21 days or less prior to the program start date: 100% cancellation fee</li>
+      </ol>
+      <br/>
+    </li>
+    <li>
+      All enrollment deferrals must be requested in writing to bootcamp@mit.edu. Deferral fees, as a percentage of published tuition rates, are as follows:
```
Comment (noisecapella) [inline bootcamp/templates/bootcamp/tac.html]: Can you make `bootcamp@mit.edu` a mailto link?

File: bootcamp/templates/bootcamp/tac.html [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+    </li>
+ </ol>
+ <strong>Ownership of Program Materials</strong>
+ <p>
+  MIT Bootcamps or other material contributors to the open enrollment programs solely own all the intellectual property
+  or have obtained the necessary rights relating to the materials used for the open enrollment programs. Title to and the
+  right to determine the disposition of any copyrightable materials first produced under the terms of Agreement solely
+  by faculty or staff of MIT shall remain with MIT and will be governed by the MIT intellectual property policies. You
+  must not copy, reproduce, republish,disassemble, decompile, reverse engineer, download, post, broadcast, transmit,
+  make available to the public, or otherwise use MIT Bootcamp content in any way except for your own personal,
```
Comment (noisecapella) [inline bootcamp/templates/bootcamp/tac.html]: Can you change `MIT Bootcamp` to `MIT Bootcamps`?

File: bootcamp/templates/bootcamp/tac.html [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+  non-commercial use. You also agree not to adapt, alter or create a derivative work from any MIT Bootcamp content
```
Comment (noisecapella) [inline bootcamp/templates/bootcamp/tac.html]: Can you change `MIT Bootcamp` to `MIT Bootcamps`?

File: bootcamp/templates/bootcamp/tac.html [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+  except for your own personal, non-commercial use. Any other use of MIT Bootcamp content requires the prior
```
Comment (noisecapella) [inline bootcamp/templates/bootcamp/tac.html]: Can you change `MIT Bootcamp` to `MIT Bootcamps`?

File: bootcamp/templates/bootcamp/tac.html [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+  The fee includes tuition, meals during the program, and course materials. The fee does not include accommodation,
+  incidental costs of the attendees, or travel costs to and from the course site, unless stated otherwise. A deposit fee as
+  set in the admission letter must be paid within 14 days of the date of acceptance (date of admissions letter). The
+  balance of the fee ([fee-deposit fee] minus any scholarships received from MIT Bootcamps) must be fully paid one
+  month before the start of the program. In the event that there is a difference between the payment due dates
+  specified above and the admissions letter, we will follow the payment due date as indicated on the admissions letter.
+  </p>
+ <strong>Cancellation and Deferral Policies</strong>
+ <p>
+     All registrations made under this agreement will be subject to MIT Bootcamp’s standard cancellation and deferral policies, including:
```
Comment (noisecapella) [inline bootcamp/templates/bootcamp/tac.html]: Can you change `MIT Bootcamp's` to `MIT Bootcamps'`?

**Comment by amir-qayyum-khan** [comment]
Done @noisecapella

**Pull request merged**